package.json

```diff
@@ -130,6 +130,8 @@
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
     "@wordpress/browserslist-config": "^4.0.0",
+    "@wordpress/dependency-extraction-webpack-plugin": "^3.2.1",
```

Comment:
Could you please explain why did you need this dependency?

Reply:
See https://github.com/mailpoet/mailpoet/pull/3778#discussion_r742827507